### PR TITLE
Copyedit ui library docs

### PR DESCRIPTION
**Pull request by kaitlynmortimer**

### CODE REVIEW

File: content/user/libraries/LibraryAnalysis.md [hunk 1]
```diff
@@ -4,47 +4,20 @@ description: "Overview of how libraries are analyzed"
...
 
->**Note:** WebSphere repackages libraries at runtime, so their SHA-1 will be different than anything known to Contrast. In order to preserve the SHA-1 during deployment, you have to set the JVM system property ```org.eclipse.jst.j2ee.commonarchivecore.ignore.web.fragment``` to ```true```. Also, any ```wsadmin``` calls must have the same parameter, as is shown here:
+<a href="assets/images/Application-libraries-tab.png" rel="lightbox" title="View libraries in an application"><img class="thumbnail" src="assets/images/Application-libraries-tab.png"/></a>
 
->```wsadmin -javaoption "-Dorg.eclipse.jst.j2ee.commonarchivecore.ignore.web.fragment=true"```
+>**Note:** WebSphere repackages libraries at runtime, so their SHA-1 is different than anything known to Contrast. To preserve the SHA-1 during deployment, set the JVM system property `org.eclipse.jst.j2ee.commonarchivecore.ignore.web.fragment` to `true`. Also, any `wsadmin` calls must have the same parameter: <br> 
+`wsadmin -javaoption "-Dorg.eclipse.jst.j2ee.commonarchivecore.ignore.web.fragment=true"`. 
 
+## Used Classes
```
[the remark this answers is not in the document]
Comment (kaitlynmortimer) [reply content/user/libraries/LibraryAnalysis.md]: @ldemidov or @m0nkey653 - Can you please advise?

File: content/user/libraries/Explore-libraries.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+View and manage libraries in the **Libraries** page, a library's details page, or the **Libraries** tabs in an application or server's details page. 
+
+## View Libraries 
+
+The libraries grid provides basic information about each library, such as: 
+
+* Letter grades
+* Applications using the library 
+* CVEs found in the library
```
[the remark this answers is not in the document]
Comment (kaitlynmortimer) [reply content/user/libraries/Explore-libraries.md]: @cixel Talked to Andria re: the terminology - sounds like it's part of a larger convo re: the UI, so we're keeping "CVEs" for now :(

File: content/troubleshooting/setup/faq/Libraries-FAQ.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Contrast identifies libraries by their SHA-1 digest, and updates library definitions periodically. As a result, Contrast might not recognize new libraries when agents report them. 
+
+> **Note:** If any library repackaging occurs for **Java** clients, which WebSphere does by default, the digest is different. To prevent repackaging, you can add the following JVM system property:
+```
+-Dorg.eclipse.jst.j2ee.commonarchivecore.ignore.web.fragment=true
+``` 
+
+## Why doesn't Contrast recognize my library as the most-recent version?
+
+Issues with library version recognition result from the way that Contrast data sources store information about a library's version. Some of the versions are formatted to include the year at the beginning, and so the sort method reads them as a more recent version. The problem should phase itself out as the industry is moving to a more normalized version convention.
```
Comment (ldemidov) [inline content/troubleshooting/setup/faq/Libraries-FAQ.md]: This is Java/maven specific.  .NET uses Nuget and Node uses npm.  Our reasons for not finding a library a slightly different.

Comment (kaitlynmortimer) [reply content/troubleshooting/setup/faq/Libraries-FAQ.md]: @ldemidov Will add a note here, and plan to address this further in the future branch/PR  (which will be coming soon!)

**Pull request merged**